lib/rucio/web/ui/static/rucio.js

```diff
@@ -306,11 +306,31 @@ RucioClient.prototype.list_accounts = function(options) {
         });
 };
 
-/* get limits for an account */
-RucioClient.prototype.get_account_limits = function(options) {
+/* get local limits for an account */
+RucioClient.prototype.get_local_account_limits = function(options) {
     check_token();
     if (options.async == null) { options.async = true; }
-    jQuery.ajax({url: this.url + '/accounts/' + options.account + '/limits',
+    jQuery.ajax({url: this.url + '/accounts/' + options.account + '/limits/local',
+                 crossDomain: true,
+                 headers: this.headers,
+                 dataType: 'text',
+                 async: options.async,
+                 success: function(data)
+                 {
+                     options.success(JSON.parse(data.replace(/Infinity/g, "-1")));
+                 },
+                 error: function(jqXHR, textStatus, errorThrown)
+                 {
+                     options.error(jqXHR, textStatus, errorThrown);
+                 }
+        });
+};
+
+/* get global limits for an account */
+RucioClient.prototype.get_global_account_limits = function(options) {
```

Comment:
Looks like this mostly has the same behaviour as the `get_local_account_limits` function, except for the API endpoint. Would it be worth making a generic `get_account_limits` a function to which we can also pass a flag/string for `local` or `global`, which then sets the API endpoint as needed? 

I think it only depends on whether we ever expect to have a different behaviour for global and local queries here. We could also make the argument that this is the old webUI and thus we don't care too much about avoiding duplication, since it's going to be replaced anyway.